client/web/src/org/settings/OrgSettingsArea.tsx

```diff
@@ -44,13 +46,16 @@ export const OrgSettingsArea: React.FunctionComponent<Props> = props => {
     // we can ignore the loading and error states in this case
     // if there is an error, we will not show the code host connections and repository screens
     // same for until the feature flag value is loaded (which in practice should be fast)
```

Comment:
I think we should change the comment as we do not ignore the loading component anymore
```suggestion
    // we can ignore the error state in this case
    // if there is an error, we will not show the code host connections and repository screens
```

Reply:
I added your change locally